src/dropdown/README.md

```diff
@@ -2,18 +2,32 @@
 
 The `vscode-dropdown` is a web component implementation of a [select element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select).
 
-## Attributes
+![Dropdown hero](/docs/assets/images/dropdown-hero.png)
+
+## Usage
```

Comment:
Do we need an example for dropdowns without a label? I know we do this in a few areas (settings, debugging) but not sure if we'd have guidance that would match that.